mkt/__init__.py

```diff
@@ -1,18 +1,19 @@
+# flake8: noqa
```

Comment:
I felt that since this file was essentially dodgy import * anyway, just noqa the entire file was fine.